python/nano/src/bigdl/nano/deps/ipex/ipex_inference_model.py

```diff
@@ -203,16 +206,25 @@ def status(self):
         return status
 
     @staticmethod
-    def _load(path, model, inplace=False):
+    def _load(path, model, input_sample=None, inplace=False):
         status = PytorchIPEXJITModel._load_status(path)
         checkpoint_path = path / status['checkpoint']
         if status["use_jit"]:
-            if status["use_ipex"]:
-                import intel_extension_for_pytorch as ipex
-            model = torch.jit.load(checkpoint_path)
-            model.eval()
-            model = torch.jit.freeze(model)
-            from_load = True
+            if status['compression'] == "bf16":
+                state_dict = torch.load(checkpoint_path, map_location='cpu')
+                if status['compression'] == "bf16":
+                    state_dict = transform_state_dict_to_dtype(state_dict, dtype="fp32")
+                model = copy.deepcopy(model)
```

Comment:
Maybe we should decide whether to  `deepcopy` model based on `inplace` parameter ?

Reply:
Have revised it, thanks